docs/quickstart/export_geometry.md

```diff
@@ -0,0 +1,53 @@
+# Export geometry
+
+Here we document how to export point clouds and meshes from nerfstudio. The main command you'll be working with is `ns-export`. Our point clouds are exported as `.ply` files and the textured meshes are exported as `.obj` files.
+
+## Install extra dependencies
+
+If you want to texture the mesh with `--unwrap-method xatlas`, then you need to install the following package shown below. We use PyTorch to rasterize the mesh and interpolate vertices and normals in the texture image.
+
+## Exporting a mesh
+
+> Poisson surface reconstruction
+
+    We default to using Poisson surface reconstruction implemented in Open3D, followed by higher resolution texturing by using a texture image and querying the NeRF. See the steps below to train your own model and export as a mesh.
+
+    1. Train nerfacto with network settings that predict normals.
```

Comment:
Add admonition stating that this will only work with models that predict normals, ie nerfacto.